src/utils/canvas.js

```diff
@@ -0,0 +1,25 @@
+
+// NOTE (pradeep): Basic canvas that renders contents at a higher resolution for clarity
+const hdCanvas = (opts = {}) => {
```

Comment:
The trick here is that we render sequence viewer canvases at a higher resolution for clairity, but then downscale them to the client's resolution to make everything fit.
You can read about this here -- https://blender.stackexchange.com/questions/105515/why-does-increasing-resolution-and-then-downsampling-to-desired-size-increase-re